docs/detections/rules-ui-create.asciidoc

```diff
@@ -263,33 +263,37 @@ For example, if a rule has an interval of 5 minutes, no additional look-back tim
 [[create-esql-rule]]
 === Create an ES|QL rule
 
-IMPORTANT: This is a placeholder for future documentation. The following content is incomplete.
+preview::[]
 
 . Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
-. To create a rule that uses ES|QL, select **ES|QL**, 
-then write a query. There are two types of ES|QL queries:
-
-.. **Aggregating query**: This is a query that uses the `STATS...BY` grouping commands. Query results cannot be matched with a particular document in Elasticsearch. For example:
+. Select **ES|QL**. 
+. Write an aggregating or non-aggregating type query:
 +
-[esql]
------
+TIP: Click the **ES|QL reference** button [book icon] for more information on ES|QL syntax. 
```

Comment:
should we specify button located in ES|QL input or add a screenshot with it?

Reply:
Yep! Added these deets in [ab28d69](https://github.com/elastic/security-docs/pull/4062/commits/ab28d69cc19c54046fda4c1e54df7dee78bf0ab6).